phpunit/block-supports/typography-test.php

```diff
@@ -283,6 +283,111 @@ public function test_should_generate_classname_for_font_family() {
 		$this->assertSame( $expected, $actual );
 	}
 
+	/**
+	 * Tests that stabilized typography supports will also apply to blocks using
+	 * the experimental syntax, for backwards compatibility with existing blocks.
+	 *
+	 * @covers ::wp_apply_typography_support
+	 */
+	public function test_should_apply_to_experimental_typography_supports() {
+		$this->test_block_name = 'test/experimental-typography-supports';
+		register_block_type(
+			$this->test_block_name,
+			array(
+				'api_version' => 3,
+				'attributes'  => array(
+					'style' => array(
+						'type' => 'object',
+					),
+				),
+				'supports'    => array(
+					'typography' => array(
+						'__experimentalFontFamily'     => true,
+						'__experimentalFontStyle'      => true,
+						'__experimentalFontWeight'     => true,
+						'__experimentalLetterSpacing'  => true,
+						'__experimentalTextDecoration' => true,
+						'__experimentalTextTransform'  => true,
+					),
+				),
+			)
+		);
+		$registry   = WP_Block_Type_Registry::get_instance();
+		$block_type = $registry->get_registered( $this->test_block_name );
+		$block_atts = array(
+			'fontFamily' => 'serif',
+			'style'      => array(
+				'typography' => array(
+					'fontStyle'      => 'italic',
+					'fontWeight'     => 'bold',
+					'letterSpacing'  => '1px',
+					'textDecoration' => 'underline',
+					'textTransform'  => 'uppercase',
+				),
+			),
+		);
+
+		$actual   = gutenberg_apply_typography_support( $block_type, $block_atts );
+		$expected = array(
+			'class' => 'has-serif-font-family',
+			'style' => 'font-style:italic;font-weight:bold;text-decoration:underline;text-transform:uppercase;letter-spacing:1px;',
+		);
+
+		$this->assertSame( $expected, $actual );
+	}
+
+	/**
+	 * Tests that stabilized typography supports are applied correctly.
+	 *
+	 * @covers ::wp_apply_typography_support
+	 */
+	public function test_should_apply_to_stabilized_typography_supports() {
```

Comment:
```suggestion
	public function test_should_apply_stabilized_typography_supports() {
```

Same a comment on the previous test.